release-please-config.json

```diff
@@ -1,9 +1,10 @@
 {
   "packages": {
     ".": {
-      "release-type": "simple",
+      "release-type": "node",
       "bump-minor-pre-major": true,
-      "versioning": "default"
+      "versioning": "default",
+      "extra-files": ["README.md"]
```

Comment:
Ok, so are you adding the readme side of this at another point?

Reply:
Good catch. Fixed!